templates/components/settings/python/settings.py

```diff
@@ -168,8 +181,8 @@ def init_anthropic():
     }
 
     Settings.llm = Anthropic(model=model_map[os.getenv("MODEL")])
-    # Anthropic does not provide embeddings, so we use FastEmbed instead
-    init_fastembed()
+    # Anthropic does not provide embeddings, so we use open Sentence Transformer models instead
```

Comment:
@tomaarsen we actually had a PR that was replacing sentence transformers (`llama-index-embeddings-huggingface@0.2.0`) with fastembed (see https://github.com/run-llama/create-llama/pull/162/files) - the reasons have been:
1. sentence transformers was a too large dependency
2. we had problems running it in the docker container of RAGapp (some pytorch issue that I forgot about)
before we replace it back - how is the situation now?